cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -245,6 +245,17 @@ firewall_tcp_ports:
   - "22/tcp"
   - "80/tcp"
   - "{{ https_enabled | ternary('443/tcp', omit) }}"
+# The following values are used in nginx to prevent undesired impact of the
```

Comment:
I think there is a use case for optionally setting all these values to nginx defaults, for the purposes of debugging or testing without any rate limiting in the mix.  Can we parameterize this in such a way that all rate limiting is turned off based on some flag?  Or alternatively, we could document in the comments what the default values are, and from there  a user could set those to turn off the rate limiting.  Ideally we could auto-inherit the defaults from the nginx upstream configuration.

Reply:
Agreed on this comment, across the board.

Reply:
It appears there are no default nginx values for these, curiously enough. Where applicable I am using info from nginx.com/blog posts about rate limit features, and also things I have tested.

I will make rate limiting opt-in only and document values in place if users want to customize from suggested values.